### ISSUE TITLE
Report Error while loading the files([object Object])

### ISSUE THREAD

**Issue by lczc1988** [rajkumar-rangaraj/MAWSFileExplorer#3]
Report Error while loading the files([object Object])
![azurediskusageexplorer](https://cloud.githubusercontent.com/assets/8097498/15345511/54cd1aaa-1ce2-11e6-893f-3a198779e245.png)

**Comment by DashleenBhandari-MSFT-zz** [comment]
I looked into the fiddler and saw that we got HTTP 500 for https://axicorp-axitradercomwebsite.scm.azurewebsites.net/DiskUsage/ during 5/18 3:50 AM GMT.

So, basically we are getting 500.121 which means that the Azure load balancer is timing out because the request DiskUsage/api/files is taking around ~240 seconds.

We have three large VMS and 43 GB of data that he is trying to identify which folder contains this large data. We can identify the folder containing large data by using PowerShell.

[Just for information – The above time-out is 4-minute idle timeout on the TCP level and it is implemented on the Azure hardware load balancer. This timeout is not configurable and this cannot be changed. One thing I want to mention is that this is idle timeout at the TCP level which means that if the connection is idle only and no data transfer happening, only then this timeout is hit. To provide more info, this will hit if the web application got the request and kept processing the request for > 4minutes without sending any data back].

Here is PowerShell Script:

$startFolder = "D:\home"

$dir = (Get-ChildItem $startFolder | Measure-Object -property length -sum -ErrorAction SilentlyContinue)
"$startFolder -- " + "{0:N2}" -f ($dir.sum / 1MB) + " MB"

$dir = (Get-ChildItem $startFolder -recurse | Where-Object {$_.PSIsContainer -eq $True} | Sort-Object)
foreach ($i in $dir)
{
    $subFolderItems = (Get-ChildItem $i.FullName | Measure-Object -property length -sum -ErrorAction SilentlyContinue)
    $i.FullName + " -- " + "{0:N2}" -f ($subFolderItems.sum / 1MB) + " MB"
}
